tests/lean/run/smt2_P_implies_Q_fail.lean

```diff
@@ -0,0 +1,5 @@
+example (P Q : Prop) : P -> Q :=
+begin
+    intros,
+    z3
```

Comment:
I agree, hadn't figured out the right place to put them yet. Maybe we should just move this code out of the standard library into a package? I don't know how far along the build tooling is.